test/refract-test.coffee

```diff
@@ -331,5 +331,28 @@ describe('Transformations • Refract', ->
         assert.equal(ast.sections[1].resources.length, 2)
       )
     )
+
+    describe('Mixed Resources and Resource Groups', ->
+      applicationAst = null
+
+      before(->
+        applicationAst = convertToApplicationAst(
+          require('./fixtures/refract-parse-result-tags.json')
+        )
+      )
+
+      it('Has two sections', ->
+        assert.strictEqual(applicationAst.sections.length, 2)
+      )
+
+      it('First section contains two resources', ->
+        assert.strictEqual(applicationAst.sections[0].resources.length, 6)
+      )
+
+      it('Second section contains two resources', ->
```

Comment:
Nit: I'd recommend in not coupling your tests descriptions so closely to implementation. If you change the fixture, you have to completely change these tests.

```coffeescript
it('has the correct sections')
it('has the correct resources')
```